src/dataworker/PoolRebalanceUtils.ts

```diff
@@ -598,24 +594,27 @@ export function generateMarkdownForRootBundle(
   });
 
   let slowRelayLeavesPretty = "";
-  slowRelayLeaves.forEach((leaf, index) => {
+  slowRelayLeaves.forEach((leaf: interfaces.SlowFillLeaf, index) => {
```

Comment:
Is this one also required? It seems like it should be OK without it.

```suggestion
  slowRelayLeaves.forEach((leaf, index) => {
```